tests/test_features/test_rankd.py

```diff
@@ -295,7 +295,20 @@ def test_rank1d_integrated_numpy(self):
         # Image similarity testing
         oz.finalize()
         self.assert_images_similar(oz)
+    
+    @pytest.mark.xfail(
+        IS_WINDOWS_OR_CONDA,
+        reason="font rendering different in OS and/or Python; see #892",
+    )
```

Comment:
@mchestnut91 @pdamodaran is this xfail necessary? It looks like this test hasn't been through CI yet without the fail, so unless I've missed something, can we remove the xfail decorator and allow the test to run as expected?

Reply:
@rebeccabilbro - I removed the xfail decorator and successfully reran the tests.

Reply:
thank you!